tests/test_ordering.py

```diff
@@ -1,27 +1,73 @@
-import re
+# -*- coding: utf-8 -*-
+
+__author__ = "svchipiga@yandex-team.ru"
```

Comment:
Can you remove the `__author__` lines from the individual files, and add an AUTHORS file at the top level?